pkg/mainflow.go

```diff
@@ -66,6 +66,7 @@ func Run(rawEnv map[string]string, namespace string) error {
 
 func printConfig(checkupConfig config.Config) {
 	log.Println("Using the following config:")
```

Comment:
Please consider taking the base config as a parameter, so if more fields will be added to it in the future - we could print them as well:
```suggestion
func printConfig(baseConfig kconfig, checkupConfig config.Config) {
	log.Println("Using the following config:")
	log.Printf("Timeout: %q", baseConfig.Timeout)
```

With this suggestion, there is no need to change the `config` package.

Reply:
DONE